### PR TITLE
Heartbeats and energy profiling support for Android

### PR DESCRIPTION
**Pull request by connorimes**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/8253)

### CODE REVIEW

File: components/util/opts.rs [hunk 1]
```diff
@@ -223,6 +226,9 @@ pub struct DebugOptions {
     /// Profile which events script tasks spend their time on.
     pub profile_script_events: bool,
 
+    /// Enable all heartbeats for profiling.
+    pub profile_heartbeats: bool,
```
Comment (mbrubeck) [inline components/util/opts.rs]: Please add this option to the `--help` output below.

**Comment by mbrubeck** [comment]
r=mbrubeck with one comment above fixed

File: components/util/opts.rs (hunk 1 above)
Comment (connorimes) [inline components/util/opts.rs]: Done. Also added missing line for `profile-script-events`.
Moved `convert-mouse-to-touch` to be in the same order as it is above.

**Comment by eefriedman** [comment]
(#7731.)

**Pull request merged**